README.md

```diff
@@ -0,0 +1,24 @@
+# Gen-AI Training for Java Developers
+
+## Project Setup
+1. Clone the repository.
+2. Edit `application.yaml` to add the following environment variables:
```

Comment:
It would be better to add it to the environment variables in your run configurations. That way, Git will not detect these as changes.